### PR TITLE
refactor(examples): modern examples with codec and merged indexer

### PR DESCRIPTION
**Pull request by IronLu233**
# Description

This PR do two things:
- merge example Indexer class into one parameter, please see: https://github.com/ckb-js/lumos/pull/445
- remove generated molecule code, use `@ckb-lumos/codec` instead

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactor (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation or Website
- [ ] Example
- [ ] Other

# How Has This Been Tested?

- Use origin test
- All examples be tested in local

### CODE REVIEW

File: packages/ckb-indexer/src/indexer.ts [hunk 1]
```diff
@@ -51,6 +51,9 @@ export class CkbIndexer implements CellProvider, TerminableCellFetcher {
   medianTimeEmitters: EventEmitter[] = [];
   emitters: IndexerEmitter[] = [];
   isSubscribeRunning = false;
+
+  constructor(ckbRpcUrl: string);
```
Comment (IronLu233) [inline packages/ckb-indexer/src/indexer.ts:55]: it says in https://github.com/ckb-js/lumos/pull/445
``` ts
import { Indexer } from '@ckb-lumos/lumos';

// before
// const indexer = new Indexer(
//   "https://testnet.ckb.dev/indexer", 
//   "https://testnet.ckb.dev/rpc",
// );

// after
const indexer = new Indexer("https://testnet.ckb.dev/rpc");
```

So when it have one parameter, the first parameter means `ckbRpcUrl`, not `ckbIndexerUrl`

File: examples/omni-lock-metamask/index.tsx [hunk 2]
```diff
@@ -49,7 +49,10 @@ export function App() {
 
     transfer({ amount: transferAmount, from: omniAddr, to: transferAddr })
       .then(setTxHash)
-      .catch((e) => alert(e.message || JSON.stringify(e)))
+      .catch((e) => {
+        console.log(e);
```
Comment (PainterPuppets) [inline examples/omni-lock-metamask/index.tsx:53]: remove console

**Review by PainterPuppets — APPROVED** [review]
LGTM

File: examples/omni-lock-metamask/index.tsx (hunk 2 above)
Comment (IronLu233) [reply examples/omni-lock-metamask/index.tsx:53]: I think it's necessary. Because the user may change the example, and put it to console takes more help.

**Review by homura — APPROVED** [review]
LGTM

**Pull request merged**